.slack/apps.json

```diff
@@ -1 +1,11 @@
-{}
+{
```

Comment:
Definitely should not be including this in the template app. Don't want anyone cloning this down to by default attempt to deploy to your personal workspace 😅